mtbf_driver/mtbf.py

```diff
@@ -205,16 +206,29 @@ def deinit(self):
         if os.path.exists(info):
             shutil.copy2(info, self.workspace)
 
+        archive_file = os.path.basename(self.archive_folder) + ".zip"
+        try:
+            os.remove(archive_file)
```

Comment:
I think this is a wonderful idea. Just patch and please see latest version, thanks.